apps/citibikebymk/citibike_by_mk.star

```diff
@@ -0,0 +1,97 @@
+load("encoding/base64.star", "base64")
+load("encoding/json.star", "json")
+load("http.star", "http")
+load("render.star", "render")
+load("schema.star", "schema")
+
+url_stat = "https://gbfs.citibikenyc.com/gbfs/en/station_information.json"
+url_bikes = "https://gbfs.citibikenyc.com/gbfs/en/station_status.json"
+resp = http.get(url_bikes)
```

Comment:
We'll need some caching around these calls to keep request rates to the API at a reasonable level.

Reply:
Thanks @matslina. Where can I find documentation on how to set up caching for API calls in my starlark project?

Reply:
Nevermind. Figured it out.